src/Term/AliasGroup.php

```diff
@@ -19,7 +19,7 @@
 class AliasGroup implements Comparable, Countable {
 
 	/**
-	 * @var string
+	 * @var string Requested and actual language of the aliases.
```

Comment:
> The documentation should say how it's supposed to be used.

Yeah. The problem here is that the documentation introduced here talks about a subset of all valid use cases. What about the concrete example I provided?

> You are nitpicking about a word.

That is your impression perhaps. I'm objecting against the meaning of the documentation.